### ISSUE TITLE
Input strings with quotes give artifacts in output

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jbeder/yaml-cpp#202]
```
The code below shows this error. (using version 0.5.1)

    YAML::Node n(YAML::Load("YAML: \"test\""));
    YAML::Emitter em;
    em << n;    
    std::string mystr(em.c_str());
    std::cout << mystr << std::endl;

What it outputs is:
YAML: !<!> test

The exclamation marks are not supposed to be there. This problem is magnified 
if you set the options to add quotes around strings in the output. In that case 
a round trip (read-write-read-write) will add more and more exclamation marks.
```

Original issue reported on code.google.com by `pweijt...@gmail.com` on 21 Apr 2013 at 8:55

**Comment by egorpugin** [comment]
Hi,

Any updates on this?

**Comment by jbeder** [comment]
No.

**Comment by jbeder** [comment]
Fixed d0250400496b6ac61f672bdad116c36c24383dd8.
